connector/failoverconnector/metadata.yaml

```diff
@@ -4,11 +4,11 @@ scope_name: otelcol/failover
 status:
   class: connector
   stability:
-    development: [metrics_to_metrics, traces_to_traces, logs_to_logs]
-  distributions: []
+    alpha: [traces_to_traces, metrics_to_metrics, logs_to_logs]
```

Comment:
You'll need to run `make generate` to make sure the component's README also gets updated.

Reply:
Ah good call thanks